### PR TITLE
Display map

### PR DESCRIPTION
**Pull request by cintiavazquez**
With this small PR I have:
-Displayed a non-interactive map (you can only zoom in and out of it) 
-If the user allows it, the map centers on user's location

### CODE REVIEW

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -7,13 +7,16 @@ export default function Layout({ children }) {
 	return (
 		<Div maxWidth="500px" margin="5vh auto 5vh auto">
 			<DivFlex
+				background="white"
```
Comment (doemser) [inline src/components/Layout/Layout.js:10]: here you are using "white" for the background color, 2 lines after this you are using a color-variable. You should be more consistent and have color-variables for all colors you are using.

**Review by doemser — APPROVED** [review]
Looks pretty good. Only some minor things.

File: src/components/Layout/Layout.js (hunk 1 above)
Comment (cintiavazquez) [reply src/components/Layout/Layout.js:10]: Thanks Dominik, I implemented the changes! :)

**Review by doemser — APPROVED** [review]
I approve it to you, but just recognized that you use `vh` for margins etc. This can be very harmful to your design on some devices.

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
-import Div from '../../UI/Div.styled';
+import DivStyled from '../../UI/DivStyled.styled';
 import DivFlex from '../../UI/DivFlex.styled';
 import Typography from '../../UI/Typography';
 import Navbar from '../Navbar/Navbar';
 
 export default function Layout({ children }) {
 	return (
-		<Div maxWidth="500px" margin="5vh auto 5vh auto">
+		<DivStyled maxWidth="500px" margin="5vh auto 5vh auto">
```
[the remark this answers is not in the document]
Comment (cintiavazquez) [reply src/components/Layout/Layout.js:8]: Oh, I've seen this just after merging... :( I'll adjust it in a future US then, thank you :)
Should I use percentages instead of `vh`? Or are percentages also problematic?

[the remark this answers is not in the document]
Comment (doemser) [reply src/components/Layout/Layout.js:8]: `%` depends on the parent-element, which could lead to similar difficulties. I usually would prefer units that are not responsive for margins and paddings. 
But in the end its up to you to test edge cases and decide which solutions are best for your design.
